Caboodle/Permissions/Permissions.ios.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CoreLocation;
+using Foundation;
+using UIKit;
+
+namespace Microsoft.Caboodle
+{
+    internal static partial class Permissions
+    {
+        static void PlatformEnsureDeclared(PermissionType permission)
+        {
+            var info = NSBundle.MainBundle.InfoDictionary;
+
+            if (permission == PermissionType.LocationWhenInUse)
+            {
+                if (!info.ContainsKey(new NSString("NSLocationWhenInUseUsageDescription")))
+                    throw new PermissionException("On iOS 8.0 and higher you must set either `NSLocationWhenInUseUsageDescription` or `NSLocationAlwaysUsageDescription` in your Info.plist file to enable Authorization Requests for Location updates!");
```

Comment:
This exception talks about iOS 8.0 - what happens on 7.0? Should there be another check somewhere or condition?

Reply:
Fixed this, it shouldn't bother checking info.plist in < 8.0